home/ivp-planning/user-interface/the-info-panel.md

```diff
@@ -9,7 +9,7 @@ The info panel displays several information about the [selected object(s)](../ma
 * _**ID:**_ the unique ID of the object which is generated when the machine is [placed ](../machines/first-steps-with-3d-object.md)in the [2D-](the-2d-panel.md) or [3D-panel](the-3d-panel.md); the ID is unchangeable and will persist until the object is [deleted](../machines/copy-and-delete-objects.md#delete-objects)
 * _**Model:**_ the model which the object uses; this information is unchangeable and independent from the object name
 * _**Name:**_ the name of the object which [can be changed](../the-tree-view/renaming-objects-and-folders.md) at any time
-* _**Position:**_ the [position ](../machines/selecting-and-moving-objects.md#move-objects)of the object relative to the current [reference point](the-grid.md#adjusting-the-reference-point) of the[ coordinate system](the-grid.md) and its [rotation](../machines/scale-and-rotate-objects.md#rotate-objects) relative to its default rotation
+* _**Position:**_ the [position ](../machines/selecting-and-moving-objects.md#move-objects)of the object relative to the current [reference point](the-grid.md#adjusting-the-reference-point) of the[ coordinate system](the-grid.md) and its [rotation](../machines/rotate-objects.md#rotate-objects) relative to its default rotation
```

Comment:
Link on "position" and "reference point" is broken. Also move the spaces out of the links here aswell